File: Desktop/ailydian-ultra-pro/docs/sdks/csharp/Examples/Quickstart.cs
using System;
using System.Threading.Tasks;
using Lydian.SDK;
using Lydian.SDK.Models;

class Program
{
    static async Task Main(string[] args)
    {
        // Initialize client
        var client = new LydianClient(new LydianConfig
        {
            ApiKey = Environment.GetEnvironmentVariable("LYDIAN_API_KEY")
        });

        // Create a city
        var city = await client.SmartCities.CreateCityAsync(new City
        {
            Name = "San Francisco",
            Country = "USA",
            Population = 873965
        });

        Console.WriteLine($"City created: {city.Id}");
    }
}

File: docs/sdks/typescript/examples/quickstart.ts
import { Lydian } from '@lydian/sdk';

// Initialize client with API key
const lydian = new Lydian({
  apiKey: process.env.LYDIAN_API_KEY,
});

async function main() {
  try {
    // Create a new city
    const city = await lydian.smartCities.createCity({
      name: 'San Francisco',
      country: 'USA',
      population: 873965,
      timezone: 'America/Los_Angeles',
    });

    console.log('City created:', city);
  } catch (error) {
    console.error('Error:', error);
  }
}

main();
